geode-gfsh/src/integrationTest/java/org/apache/geode/management/internal/cli/GfshParserAutoCompletionIntegrationTest.java

```diff
@@ -381,7 +381,7 @@ public void testObtainHintForData() {
     String hintArgument = "data";
     String hintsProvided = gfshParserRule.getCommandManager().obtainHint(hintArgument);
     String[] hintsProvidedArray = hintsProvided.split(lineSeparator());
-    assertThat(hintsProvidedArray.length).isEqualTo(15);
+    assertThat(hintsProvidedArray.length).isEqualTo(16);
```

Comment:
This is an existing test measuring our parsing of gfsh hints. Because a new hint was added for clear, it's now finding one more hint than it was before, but this is an expected difference so I've just updated the tests assertion to reflect that.